### ISSUE TITLE
Block-scoped declarations issue

### ISSUE THREAD

**Issue by ThatOneGamer999** [beakerbrowser/bkr#18]
Trying to run the "bkr" command will always result in this issue:

```
/usr/local/lib/node_modules/bkr/node_modules/dat-dns/index.js:54
    let key, ttl
    ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/bkr/build/lib/opts.js:74:15)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)

```

EDIT: My OS is Ubuntu 16.04 LTS using Linux 4.4.0-64-generic

**Comment by pfrazee** [comment]
What version of node are you running?

**Comment by pfrazee** [comment]
(Sorry for the late response)

**Comment by pfrazee** [comment]
Actually NM, dat-dns isnt supposed to be using es6 features. I published dat-dns@1.0.4. Reinstall bkr (make sure dat-dns gets updated) and lmk if you still have this problem
